components/Article/Article.styled.js

```diff
@@ -0,0 +1,55 @@
+import styled from "styled-components";
+import Link from "next/link";
+import { css } from "styled-components";
+
+const StyledArticle = styled.article`
+  margin: 0 25px 25px 25px;
+
+  position: relative;
+  background-color: white;
+  border: none;
+  border-radius: var(--border-radius);
+`;
+
+const StyledDetailsLink = styled(Link)`
+  height: 35px;
+  position: absolute;
+  right: 10px;
+  bottom: 10px;
+  border-style: none;
+  color: var(--tertiary-color);
+`;
+
+const StyledArticleImageContainer = styled.div`
+  position: relative;
+  width: 100%;
+  height: 30vh;
+  background-color: var(--article-card);
+  border-radius: var(--border-radius) var(--border-radius) 0 0;
+`;
+
+const StyledList = styled.ul`
+  list-style: none;
+  margin: 0;
+  padding: 0 0 15px 20px;
+`;
```

Comment:
I guess you don't need the `margin` - please try out in the dev tools 

```suggestion
const StyledList = styled.ul`
  list-style: none;
  padding: 0 0 15px 20px;
`;
```

Reply:
the `ul` tag  has a margin top + bottom of 16 px. I had to "reset" this